rubrics/impl/src/main/bundle/rubricsMessages.properties

```diff
@@ -28,6 +28,7 @@ criterion_description=Criterion Description
 preview_rubric=Preview Rubric
 point_override_details=Fine tune points by entering a number and overriding the selected rating points
 criterion_comment=Leave a comment about this criterion for the student
+criterion_comment_student=Professor's comment about this criterion
```

Comment:
I suggest using "Instructor" comments instead of "Professor's"

"Instructor comment about this criterion"